.electron-vue/marktextEnvironment.js

```diff
@@ -0,0 +1,32 @@
+const GitRevisionPlugin = require('git-revision-webpack-plugin')
+const { version } = require('../package.json')
+
+const getEnvironmentDefinitions = function () {
+  const gitRevisionPlugin = new GitRevisionPlugin()
+  const isOfficialRelease = !!process.env.MARKTEXT_IS_OFFICIAL_RELEASE
```

Comment:
I forget how to set the `MARKTEXT_IS_OFFICIAL_RELEASE ` variable. where to set it?

Reply:
You have to set the environment variable `MARKTEXT_IS_OFFICIAL_RELEASE` to not null (e.g. `export MARKTEXT_IS_OFFICIAL_RELEASE=1`). It's used to not add the commit hash to the version string.